react-hyf/README.md

```diff
@@ -0,0 +1,32 @@
+## React UI Names App
```

Comment:
Great to add an explanation on how to start the project in the README, very useful for other developers :) (and for me)